src/Microsoft.DotNet.ProjectModel/ProjectPath.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    internal static class ProjectPath
```

Comment:
The name is a little bit to generic. I look at this and think of a structure represents a path. It acts more like a helper class. So how about `ProjectPathHelper`?